event_registration_partner_unique/readme/INSTALL.rst

```diff
@@ -0,0 +1,2 @@
+#. Bear in mind, for this module to show its potential, it is important to
+   install `partner_event` module alongside it.
```

Comment:
That module is already a dependency for this one. Better extend the usage section with some clues on how to use them altogether.

Reply:
@chienandalu INSTALL.rst file removed from readme. Usage explanation section has been extended. Please review